### PR TITLE
Niryo ned

### PR DESCRIPTION
**Pull request by Pauline-Odet**
I added to ros robots the new Niryo Robot, Ned

### CODE REVIEW

File: _posts/2021-02-15-niryo-ned.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- 'co-bot'
+- 'dynamixel'
+- 'education'
+- 'melodic'
+- 'moveit'
+- 'raspberry pi'
+- 'research'
+twitter_text: Checkout the Ned
+website: https://niryo.com/product/ned/
+wiki_homepage: http://wiki.ros.org/niryo_ned
```
Comment (tfoote) [inline _posts/2021-02-15-niryo-ned.md]: This wiki page does not appear to exist.

**Review by tfoote — CHANGES_REQUESTED** [review]
This looks like a nicely layed out page. However I noticed that the wiki homepage linked for the robot does not appear to exist. That should have the ROS API documentation and instructions for how to integrate with the robot.

File: _posts/2021-02-15-niryo-ned.md (hunk 1 above)
Comment (Pauline-Odet) [reply _posts/2021-02-15-niryo-ned.md]: Thank you for your answer !
 Actually, we just created the wiki page and it has not been accepted yet. 
I'll let you know when the wiki page is online !

Comment (Pauline-Odet) [reply _posts/2021-02-15-niryo-ned.md]: Hi @tfoote, I just updated the wiki homepage, which is now online ! My fork is ready to be merged now

File: _posts/2021-02-15-niryo-ned.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- 'co-bot'
+- 'dynamixel'
+- 'education'
+- 'melodic'
+- 'moveit'
+- 'raspberry pi'
+- 'research'
+twitter_text: Checkout the Ned
+website: https://niryo.com/product/ned/
+wiki_homepage: http://wiki.ros.org/ned
```
Comment (tfoote) [inline _posts/2021-02-15-niryo-ned.md]:
```suggestion
wiki_homepage: http://wiki.ros.org/Robots/ned
```
Fixing url to follow our standards for wiki page naming.

**Review by tfoote — APPROVED** [review]
Thanks for iterating. Note that to follow our naming convention I moved your page to Robots/ned not just ned at the top.

**Pull request merged**